src/java.base/share/native/libjli/manifest_info.h

```diff
@@ -138,6 +140,12 @@
 #define ZIP64_LOCOFF(b) LL(b, 8)      /* offset of zip64 end */
 #define ZIP64_LOCTOT(b) LG(b, 16)     /* total number of disks */
 
+/*
+ * Macros for getting Extensible Data Fields
+ */
+#define ZIPEXT_HDR(b) SH(b, 0)      /* Header ID */
```

Comment:
How about naming the macros as ZIP64EXT_HDR and ZIP64EXT_SIZ?

Reply:
My thinking was that all extensible data fields start with a header with an id and a length, and these macros are used to iterate through the extra fields to find the zip64 extended information extra field, so these macros aren't zip64-specific.

Reply:
Seem reasonable, thanks.